module-1/pandas-project/your-code/PandasProject.ipynb

```diff
@@ -0,0 +1,990 @@
+{
```

Comment:
1. It's a good practise to explain,at the beginning of the notebook ,the kind of analysis you are going to run and main characteristics of the dataset you have chosen. 
2. In real world analysing missing values is an anylisis itself. I mean removing missing values or fill them with 0 is not always the best solution. It depends on the analysis that will be performed with the dataset. You can drop missing values for example if the nulls are a small proportion of the total number of records. A good way of having an overview is the info function. 
3. Remember to show only the head or the tail of the dataframe as needed. This generates a cleaner and more visual notebook. 
4. At the end of your notebook you have created a dataframe called null_region, what is the sense? What do you want to do? 
5. null_Region = null_Region.drop(nullRegion, axis=0). Do you understand the error this raises? Python does not recognise nullRegion as a column of your null_region dataframe. Do you understand the different ways of accesing dataframe values?